app/controllers/concerns/wizard_steps.rb

```diff
@@ -0,0 +1,66 @@
+module WizardSteps
+  extend ActiveSupport::Concern
+
+  included do
+    class_attribute :wizard_class
+    before_action :load_wizard, :load_current_step, except: %i[index completed]
+  end
+
+  def index
+    redirect_to step_path(wizard_class.first_key)
+  end
+
+  def show
```

Comment:
there was a reason for this, I will try to remember...

Reply:
It is so we can use diary_step_path for both show and update

Reply:
I think

Reply:
I think it's probably because the `edit` action by default comes with the `/edit` suffix in the url. But that can be changed in the `routes.rb` file, I believe 🤔